yarn.lock

```diff
@@ -14155,6 +14147,14 @@ redux@^4.0.1:
     loose-envify "^1.4.0"
     symbol-observable "^1.2.0"
 
+redux@^4.0.2:
```

Comment:
might want to bump redux to 4.0.2 in desktop too to avoid dup

Reply:
bumped